bootstrap_2.3.1/css/bootstrap.css

```diff
@@ -223,10 +223,11 @@ textarea {
 
 body {
   margin: 0;
-  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
-  font-size: 14px;
-  line-height: 20px;
-  color: #333333;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 13px;
+  line-height: 2.4px;
+  letter-spacing: 0.24;  
+  color: @primary_color;
   background-color: #ffffff;
```

Comment:
I think `bootstrap_2.3.1/css/bootstrap.css` should be reverted back.  It's a CSS file and not LESS file (so @primary_color won't work) and it's only used in our API docs: (https://www.bcjobs.ca/api/) for legacy reasons. Should these styles be moved somewhere else? If not, let's just revert back.